app/controllers/admin/topics_controller.rb

```diff
@@ -305,6 +305,52 @@ def assign_team
     end
   end
 
+  def refresh_list
+#    @status = params[:status]
+#    get_tickets if params[:status].present?
+
+    @status = params[:status] || "pending"
+    if current_user.is_restricted? && teams?
+      topics_raw = Topic.all.tagged_with(current_user.team_list, :any => true)
```

Comment:
Hmm I am not sure, I just repeated how @jprberlin did it originally.  I will try it.